framework/helpers/BaseConsole.php

```diff
@@ -1019,4 +1019,59 @@ public static function endProgress($remove = false, $keepPrefix = true)
         self::$_progressEtaLastDone = 0;
         self::$_progressEtaLastUpdate = null;
     }
+
+    public static function table($headers, $rows, $span = '+', $separator = '-', $conector = '|')
```

Comment:
It could be that table is drawn using different set of chars:

https://en.wikipedia.org/wiki/Box-drawing_character